Core/Core/Configuration/Config/AgreementConfig.swift

```diff
@@ -12,6 +12,7 @@ private enum AgreementKeys: String {
     case tosURL = "TOS_URL"
     case cookiePolicyURL = "COOKIE_POLICY_URL"
     case dataSellContentURL = "DATA_SELL_CONSENT_URL"
+    case eulaURL = "EULA_URL"
```

Comment:
As you are here, could you please make the changes that `AgreementKeys` implements `RawStringExtractable` so we have same pattern with all config parsing.